pkg/cmd/repo/view/view.go

```diff
@@ -170,9 +169,8 @@ func viewRun(opts *ViewOptions) error {
 		if err != nil {
 			return fmt.Errorf("error rendering markdown: %w", err)
 		}
-		readmeContent = emoji.Parse(readmeContent)
 	} else {
-		readmeContent = emoji.Parse(readme.Content)
```

Comment:
This change disables emoji support in non-Markdown documents, but I'm okay with that.